src/Config.php

```diff
@@ -308,16 +308,22 @@ protected function parseGeneral()
                 // as a regex-like string, and we switched to an array. If we find the old style, fall back to the defaults.
                 unset($general['accept_file_types']);
             }
-            // accept uppercase and lowercase file extensions
-            $general['accept_file_types'] = array_unique(
-                array_merge(
-                    $general['accept_file_types'],
-                    array_map(function ($extension) {
-                        return strtolower($extension);
-                    }, $general['accept_file_types']),
-                    array_map(function ($extension) {
-                        return strtoupper($extension);
-                    }, $general['accept_file_types'])
+
+            // Just.. Say no to these.
```

Comment:
Say yes to these and no to any others

Reply:
That's not feasible.. the entire idea about the `accept_file_types` is that it _is_ the whitelist of acceptible filetypes. We can impossibly list all the extensions that people _might_ want to put in their own lists. Simply not feasible. 

This actively _removes_ a few known bad actors, as a minor extra layer to guard against mishaps. Just like in our `allowed_tags` we let the user set which tags are allowed, but regardless we _remove_ `<script>` from that list. Same thing applies here.

Reply:
(1)

@JarJak is correct, the resulting list is a list of file types that _are_ allowed. A better comment would be something like:

```
// Remove problematic file types
```

or maybe:

```
// Remove blacklisted file types
```

(2)

The logic is incomplete: it removes `php`, but not `PHP`. You can add `PHP` to `accept_file_types` and then upload `php` files.

Reply:
(2) it can just do case-insensitive check

Reply:
In my latest version of this PR, this is a non-issue, because it's handled correctly :-)